eclipse-collections-code-generator/src/main/resources/impl/map/immutable/immutablePrimitivePrimitiveMapKeySet.stg

```diff
@@ -2,7 +2,7 @@ import "primitiveImmutableKeySets.stg"
 
 skipBoolean() ::= "true"
 
-targetPath() ::= "org/eclipse/collections/impl/map/mutable/primitive"
+targetPath() ::= "org/eclipse/collections/impl/map/immutable/primitive"
```

Comment:
This cannot be moved as it will break serialization.

Reply:
reverted